src/styles/gommette/components/_go-multiselect.scss

```diff
@@ -53,7 +55,7 @@ fieldset[disabled] .multiselect {
 .multiselect__input,
 .multiselect__single {
   font-family: inherit;
-  font-size: 16px;
+  font-size: 14px;
   touch-action: manipulation;
```

Comment:
Is not you, but touch-action: manipulation is not available on Safari.

Reply:
I don't know what it does on other browsers, do you think I should remove it?

Reply:
I will let it as I don't know what it does exactly in our case. Do you agree?